Sources/SwiftProtobuf/TextFormatTypeAdditions.swift

```diff
@@ -15,46 +15,54 @@
 
 import Foundation
 
-//
-// Messages
-//
+/// Text format encoding and decoding methods for messages.
 public extension Message {
-    /// Serializes the message to the Protocol Buffer text serialization format.
-    ///
-    /// - Throws: an instance of `TextFormatEncodingError` if encoding fails.
-    public func textFormatString() -> String {
-        var visitor = TextFormatEncodingVisitor(message: self)
-        if let any = self as? Google_Protobuf_Any {
-            any._storage.textTraverse(visitor: &visitor)
-        } else {
-            try! traverse(visitor: &visitor)
-        }
-        return visitor.result
+  /// Returns a string containing the Protocol Buffer text format serialization
+  /// of the message.
+  ///
+  /// Unlike binary encoding, presence of required fields is not enforced when
```

Comment:
So we're little inconsistent, you mention isInitialized here, but not no init(textFormatString:extensions:).  How about we drop all references from isInitialized in the TextFormat and JSON and just make the phrasing in binary clear that it is specific to that one format?